assets/js/solcast.webflow.script.js

```diff
@@ -0,0 +1,101 @@
+// Splide Scripts
+document.addEventListener('DOMContentLoaded', Splide () {
```

Comment:
Not sure this is correct - is Splide() a function?

Should this be `document.addEventListener('DOMContentLoaded', function() { ...` ?